src/_configure_plextrac.sh

```diff
@@ -207,3 +207,13 @@ function create_volume_directories() {
   fi
 }
 
+function configure_waf() {
+  title "Create directory and file for customer curated WAF rules"
+  info "Validating directory and file for customer curated WAF rules"
+  debug "Ensuring directory and file exist for WAF."
+  if test -f "${PLEXTRAC_HOME}/volumes/naxsi-waf/customer_curated.rules"; then
+    debug "WAF Config already exists"
+  else
+    mkdir -vp "${PLEXTRAC_HOME}/volumes/naxsi-waf"
+    echo "## Custom WAF Rules Below" > ${PLEXTRAC_HOME}/volumes/naxsi-waf/customer_curated.rules
+}
```

Comment:
If you want the customer to be able to currate WAF rules, I think we can remove this logic entirely in favor of a manual step(s) we can provide on request. Something along the lines of:

```shell
su - plextrac
mkdir -vp ~/volumes/naxsi-waf
echo "## Custom WAF Rules Below" > ${PLEXTRAC_HOME}/volumes/naxsi-waf/customer_curated.rules
vi docker-compose.override.yml

---
services:
  plextracnginx:
    volumes:
      - naxsi_waf:/etc/nginx/conf.d/customer_curated.rules:r
volumes:
  naxsi_waf:
    driver: local
    driver_opts:
      type: "none"
      o: "bind"
      device: "${PLEXTRAC_HOME:-.}/volumes/naxsi-waf/customer_curated.rules"
---
docker compose -f docker-compose.yml -f docker-compose.override.yml up -d plextracnginx --force-recreate
```

Reply:
I'd like to just have this populated with a comment at plextrac configure. I could add logic to look for a customer_curated.rules file in the nginx_init.sh script and if it is there combine the files, and if not then just rename the rules file I include in the image build.  Whatever you think is best is fine by me dude!